docs/tables/aws_securityhub_finding.md

```diff
@@ -149,6 +150,21 @@ where
   workflow_state = 'ASSIGNED';
 ```
 
+### List findings with Workflow Status (`worfklow_status`) NOTIFIED. (All Workflow Status: NEW, NOTIFIED, SUPPRESSED, RESOLVED)
```

Comment:
```suggestion
### List findings with workflow status NOTIFIED
```

We generally don't use periods in our descriptions. And for the possible statuses, would it be better to have them listed in the description, e.g., `The workflow status of a finding. Possible statuses are NEW, NOTIFIED, SUPPRESSED, RESOLVED.`

This helps us to avoid updating the statuses in 2 places if AWS ever changes them or adds one.